images/stackstorm/Dockerfile

```diff
@@ -80,8 +80,16 @@ COPY bin/install.sh /install.sh
 RUN /install.sh --tag=${ST2_TAG} --st2=${ST2_VERSION} --st2web=${ST2WEB_VERSION} --st2mistral=${ST2MISTRAL_VERSION} --user=${CIRCLE_PROJECT_USERNAME} --repo=${CIRCLE_PROJECT_REPONAME} --buildurl=${CIRCLE_BUILD_URL} --sha=${CIRCLE_SHA1}
 
 # Install chatops and disable unless entrypoint.d file is present
-# Install st2-chatops with Node.js v6 requirement
-RUN curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash - && sudo apt-get install -y st2chatops && echo manual | sudo tee /etc/init/st2chatops.override
+# Install st2-chatops with Node.js v6/v10 requirement
+RUN if [ -n "${ST2_VERSION}" ]; then \
+        if [ $(printf "3.0\n${ST2_VERSION}\n" | sort -V | head -n 1) = "3.0" ]; then \
```

Comment:
Please add a comment which describes what this line does (aka checks if version if >= 3.0 and if it is, it returns 3.0 and installs Node.js 10.x for StackStorm >= 3.0 and 6.x for < 3.0).